.github/workflows/create-env.yaml

```diff
@@ -16,11 +16,16 @@ jobs:
     name: Build & Push
     runs-on: ubuntu-20.04
     env:
-      IMAGE_VERSION: '2.2.1'
+      IMAGE_VERSION: '3.0.0'
       IMAGE_NAME: create-env
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+      with:
+        platforms: arm64
```

Comment:
nit: seems it should be moved in arm64 PR?

Reply:
Right! Removed it!